src/db.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+
+const connectDB = async () => {
+  try {
+    await mongoose.connect(process.env.MONGO_URI, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    });
+    console.log('✅ Connected to MongoDB');
+  } catch (err) {
+    console.error('❌ Error connecting to MongoDB:', err.message);
+    process.exit(1); // Exit the process if the connection fails
+  }
```

Comment:
This comment does not add anything (IMO), because from reading  `process.exit(1);` we can already know what this will do. 

So please remove this comment, or write me if you disagree